saleor/graphql/channel/tests/test_channel_update_mutations.py

```diff
@@ -126,9 +127,8 @@ def test_channel_update_mutation_with_invalid_slug(
     content = get_graphql_content(response)
 
     # then
-    error = content["data"]["channelUpdate"]["channelErrors"][0]
-    assert error["field"] == "slug"
-    assert error["code"] == ChannelErrorCode.INVALID.name
+    channel_data = content["data"]["channelUpdate"]["channel"]
+    assert channel_data["slug"] == slugify(slug)
```

Comment:
We should rename this test.

Reply:
The name of test is `test_channel_update_mutation_with_invalid_slug` and in this case im testing update channel with invalid slug. IMO this name is good, any sugestions how i should name it?

Reply:
Maybe `test_channel_update_mutation_slugify_slug_field` ?